includes/admin/views/metaboxes/view-order-details.php

```diff
@@ -187,14 +190,26 @@
 							<em><?php _e( 'For internal use only, not visible to the customer.', 'lifterlms' ); ?></em>
 						</label>
 
-						<button id="llms-save-remaining-payments" class="button button-primary button-large"><?php _e( 'Save', 'lifterlms' ); ?></button>
+						<button id="llms-save-remaining-payments" name="fake" class="button button-primary button-large"><?php _e( 'Save', 'lifterlms' ); ?></button>
```

Comment:
Is the `name` attribute required?

Reply:
no that's a mistake, I gave it a name to help identify it when debugging and forgot to remove it.